test/extended/images/rails_ex.go

```diff
@@ -0,0 +1,83 @@
+package images
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"k8s.io/kubernetes/test/e2e"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("images: parallel: test ruby images with rails-ex repo", func() {
```

Comment:
@bparees can we make this a Factory for producing these test cases? How much they will differ? Maybe we can just generate them from some config struct ;-)